tests/test_api.py

```diff
@@ -1066,7 +1066,7 @@ def test_set_status_list(self):
         entity = 'Task'
         entity_id = self.task['id']
         field_name = 'sg_status_list'
-        pos_values = ['rdy', 'fin']
+        pos_values = ['wtg', 'fin']
```

Comment:
`rdy` is not a status field value by default anymore, so we need to fix that test.